devenv/lib/rosetta.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import os
+
+from devenv.constants import DARWIN
+from devenv.constants import MACHINE
+from devenv.lib import proc
+
+
+def ensure() -> None:
+    if not (DARWIN and (MACHINE == "arm64")):
+        return
+
+    # this file doesn't exist if rosetta 2 isn't installed
```

Comment:
actually just provisioning a mac arm UTM VM as we speak to verify my guess

(it's really hard to uninstall rosetta after it's installed)